Scripts/Generator/MapLoader.cs

```diff
@@ -91,11 +91,14 @@ public ShaderMaterial FloorMaterial
     private RegionLabel[,] regionMap;
 
     private Level level;
+    private Player player;
 
     public override void _Ready()
     {
         ready = true;
         level = GetTree().Root.GetNodeOrNull<Level>("Level");
+        player = level.GetNodeOrNull<Player>("Player");

```

Comment:
This one probably should use `player = level?.GetNodeOrNull<Player>("Player")` since the level may be null

Reply:
I feel that if level is null we should let it crash?

Reply:
Level can be null in the preview for some reason

Reply:
But I think that's in the subcomponents of MapLoader, like EnemySpawner.

Reply:
So probably should be ok to do `GetTree().Root.GetNode<Level>("Level")` only, cos MapLoader is supposed to be in the scene tree from the get go.